core/application/impl/app_configuration_impl.cpp

```diff
@@ -352,7 +343,10 @@ namespace kagome::application {
     po::options_description desc("General options");
     desc.add_options()
         ("help,h", "show this help message")
-        ("verbosity,v", po::value<int>(), "Log level: 0 - trace, 1 - debug, 2 - info, 3 - warn, 4 - error, 5 - critical, 6 - no log")
+        ("log,l", po::value<std::string>(),
+          "Sets a custom logging filter. Syntax is `<target1>=<level1>,<target2>=<level2>`, e.g. -lkagome=info,libp2p=off.\n"
```

Comment:
We should be able to execute queries like that:
`kagome --dev -llibp2p=debug -lruntime=trace`

Reply:
Changed